src/ui/Titles/H2.tsx

```diff
@@ -1,14 +1,25 @@
-import { PropsWithChildren } from 'react'
 import styled from 'styled-components'
+import { colors } from 'styles/colors'
```

Comment:
"styles/colors" is not a library, it should be next to "ui/Titles", not next to "styled-components".

Reply:
done